### ISSUE TITLE
Rust toolchain `nightly-2020-06-10` causing block validation in Windows to fail.

### ISSUE THREAD

**Issue by hansieodendaal** [tari-project/tari#2022]
**Describe the bug**
Commit df95cee73812689bbae77bfb547c1d73a49635d4 introduced a bug in Windows builds that resulted in blocks 9182, 30335, 33923 and 34947 failing validation tests. e,g,:
```
2020-06-18 11:39:51.632946300 [c::bn::comms_interface::inbound_handler] WARN  Block #33923 
   (3a541cf21e19650697d8d785ba9b5f38e86868af94a7fca4adec3c3bcf93aef1) validation failed: ValidationError { source: 
   TransactionError(ValidationError("Sum of inputs and outputs did not equal sum of kernels with fees")) }
```
This originates from `fn validate_kernel_sum(...)` in `aggregated_body.rs`, as `fn sum_kernels(...)` returns the wrong result.
The problem is caused by updating Rust toolchain to `nightly-2020-06-10` from `nightly-2020-01-08`.

**To Reproduce**
Steps to reproduce the behaviour:
1. Ceck out commit hash ab70a86f5f17b85fa078194896a431bfbdc90246 from `development` branch dated `2020/06/24 12:22:24`
1. Run `cargo test -p tari_core --test regression_tests`

**Expected behaviour**
When using Rust toolchain `nightly-2020-06-10` all tests should pass:
```
running 4 tests
test test_block_33923 ... ok
test test_block_34947 ... ok
test test_block_30335 ... ok
test test_block_9182 ... ok
```

**Screenshots**
n/a

**Desktop (please complete the following information):**
 - Windows 10

**Smartphone (please complete the following information):**
 - n/a

**Additional context**
n/a

**Comment by stringhandler** [comment]
@hansieodendaal is this still occuring?

**Comment by hansieodendaal** [comment]
Hi @mikethetike, fixed by #2035. It can be closed.
